frontend/src/components/Search.js

```diff
@@ -33,5 +33,6 @@ const Wrapper = styled.div`
   grid-template-rows: min-content min-content min-content min-content 1fr;
   Button {
     background-color: #008080;
+    border-radius: 0px;
```

Comment:
0 pixel? I guess you could just delete it then

Reply:
Button has a fixed radius. Have to overwrite it.